### ISSUE TITLE
We need to create a job that periodically deletes test accounts github repos

### ISSUE THREAD

**Issue by ldimaggi** [fabric8io/fabric8-test#700]
If we don't do this - we will have NNNN repos in the github account.

**Comment by ppitonak** [comment]
I agree that we should do this but I would be very careful when to do it...do we want to delete it immediately after test finishes? We do remove space at the end of test.

We can use `ee_tests/clean_test_repos.sh`, I'm using it to remove old repos in my private account but it might need small changes (e.g. it requires a GitHub token and requires an explicit confirmation)

**Comment by ldimaggi** [comment]
I would opt to NOT have each repo deleted at the end of each job as there may be instances where we will want to examine the repo to debug failures.

I think that we should delete the repo's periodically and on demand. Perhaps once a week?

**Comment by ppitonak** [comment]
This needs to be done:

* [ ] generate a new token for `osiotestmachine` GitHub account with scope allowing to remove repositories
* [ ] store this token in Jenkins credentials store
* [ ] update script `clean_test_repos.sh` so that it can run in non-interactive mode
* [ ] create a new Jenkins job that will run periodically and will remove repos using `clean_test_repos.sh` script

We definitely cannot remove all repositories because it would delete also repositories for jobs that are still running. Do we want to keep repos for e.g. one month?
